app/controllers/settings/password.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  actions: {
+    changePassword(passwordData) {
+      this.set('isLoading', true);
+      let payload = {
+        'data': {
+          'old-password' : passwordData.passwordCurrent,
+          'new-password' : passwordData.passwordNew
+        }
+      };
+      this.get('loader')
+        .post('/auth/change-password', payload)
+        .then(() => {
+          this.get('notify').success(this.l10n.t('Password updated successfully'));
+        })
+        .catch(error => {
+          error = (error.error ? error.error : 'Unexpected error. Password did not change.');
+          this.get('notify').error(this.l10n.t(error));
```

Comment:
You can't pass like this to `this.l10n.t`. You need to mark it explicitly. `this.l10n.t('Unexpected error. Password did not change.')`.